server/webapp/WEB-INF/vm/admin/server_version.txt.vm

```diff
@@ -0,0 +1 @@
+{"go_version":"16.6.0","go_build_number":"3341", "git_sha":"1236015adabbfdf88a786f9e0d66aabd071c1c12"}
```

Comment:
This must be generated only while packaging.
Check-in this file with an empty json